### PR TITLE
allow spaces and commas in tag values

### PR DESCRIPTION
**Pull request by jpslav**
allow spaces and commas in tag values

### CODE REVIEW

File: lib/openstax/aws/tag.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class Tag
     attr_reader :key, :value
 
     AWS_TAG_KEY_REGEX = /\A[\w\+\-\=\.\:\/\@]{1,128}\z/
-    AWS_TAG_VALUE_REGEX = /\A[\w\+\-\=\.\:\/\@]{0,256}\z/
+    AWS_TAG_VALUE_REGEX = /\A[\w\+\-\=\.\:\/\@ \,]{0,256}\z/
```
Comment (TomWoodward) [inline lib/openstax/aws/tag.rb]: isn't most of this escaping unnecessary? can it not be:
```suggestion
    AWS_TAG_VALUE_REGEX = /\A[\w\-\.\/+=:@, ]{0,256}\z/
```

Comment (jpslav) [reply lib/openstax/aws/tag.rb]: Changed.

Comment (TomWoodward) [reply lib/openstax/aws/tag.rb]: isn't the `.` one of the ones you _do_ want escaped?

Comment (jpslav) [reply lib/openstax/aws/tag.rb]: No.  I couldn't find a Ruby description that said this but I did find [this](https://www.regular-expressions.info/charclass.html) that said:

>In most regex flavors, the only special characters or metacharacters inside a character class are the closing bracket ], the backslash \, the caret ^, and the hyphen -. The usual metacharacters are normal characters inside a character class, and do not need to be escaped by a backslash. To search for a star or plus, use [+*]. Your regex will work fine if you escape the regular metacharacters inside a character class, but doing so significantly reduces readability.  

And here's the Ruby proof: https://rubular.com/r/7qhykgBJVMkjak

**Pull request merged**